From 0158da2b8aa66c93fbec3da0cef163c79221e29d Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <panto@antoniou-consulting.com>
Date: Wed, 30 Jan 2013 20:11:05 +0200
Subject: [PATCH 098/184] cape-dvi: Switch all DVI capes to using the TFTP410
 panel

Now that TFP410 works reasonably well switch to using it.

Signed-off-by: Pantelis Antoniou <panto@antoniou-consulting.com>
---
 firmware/capes/cape-bone-dvi-00A0.dts |   50 ++-------------------------------
 firmware/capes/cape-bone-dvi-00A1.dts |   50 ++-------------------------------
 firmware/capes/cape-bone-dvi-00A2.dts |   50 ++-------------------------------
 3 files changed, 9 insertions(+), 141 deletions(-)

diff --git a/firmware/capes/cape-bone-dvi-00A0.dts b/firmware/capes/cape-bone-dvi-00A0.dts
index 293b655..92fc6be 100644
--- a/firmware/capes/cape-bone-dvi-00A0.dts
+++ b/firmware/capes/cape-bone-dvi-00A0.dts
@@ -83,57 +83,13 @@
 				};
 			};
 
-			/* Settings for DVI 1024x768@60 */
+			/* use the DVI panel now */
 			panel {
-				compatible = "tilcdc,panel";
+				compatible = "tilcdc,tfp410";
 				pinctrl-names = "default";
 				pinctrl-0 = <&bone_dvi_cape_dvi_00A0_pins>;
 				ti,power-gpio = <&gpio1 7 0x0>;
-				panel-info {
-					ac-bias           = <255>;
-					ac-bias-intrpt    = <0>;
-					dma-burst-sz      = <16>;
-					bpp               = <16>;
-					fdd               = <0x80>;
-					tft-alt-mode      = <0>;
-					stn-565-mode      = <0>;
-					mono-8bit-mode    = <0>;
-					sync-edge         = <0>;
-					sync-ctrl         = <1>;
-					raster-order      = <0>;
-					fifo-th           = <0>;
-				};
-				display-timings {
-					native-mode = <&timing1>;
-					timing0: 1024x768 {
-						hactive         = <1024>;
-						vactive         = <768>;
-						hback-porch     = <80>;
-						hfront-porch    = <48>;
-						vback-porch     = <15>;
-						vfront-porch    = <3>;
-						hsync-len       = <32>;
-						vsync-len       = <4>;
-						clock-frequency = <56000000>;
-						hsync-active	= <0>;
-						vsync-active	= <0>;
-					};
-					/* 1280 x 720 @ 60 Hz  Reduced blanking VESA CVT 0.92M9-R */
-					timing1: 1280x720-60 {
-						hactive         = <1280>;
-						hfront-porch    = <48>;
-						hsync-len       = <32>;
-						hback-porch     = <80>;
-						vactive         = <720>;
-						vfront-porch    = <3>;
-						vsync-len       = <5>;
-						vback-porch     = <13>;
-						clock-frequency = <64000000>;
-						hsync-active	= <0>;
-						vsync-active	= <0>;
-					};
-
-				};
+				i2c = <&i2c2>;
 			};
 
 			fb {
diff --git a/firmware/capes/cape-bone-dvi-00A1.dts b/firmware/capes/cape-bone-dvi-00A1.dts
index e706b93..ed9099c 100644
--- a/firmware/capes/cape-bone-dvi-00A1.dts
+++ b/firmware/capes/cape-bone-dvi-00A1.dts
@@ -93,57 +93,13 @@
 				};
 			};
 
-			/* Settings for DVI 1024x768@60 */
+			/* use the DVI panel now */
 			panel {
-				compatible = "tilcdc,panel";
+				compatible = "tilcdc,tfp410";
 				pinctrl-names = "default";
 				pinctrl-0 = <&bone_dvi_cape_dvi_00A1_pins>;
 				ti,power-gpio = <&gpio1 7 0x0>;
-				panel-info {
-					ac-bias           = <255>;
-					ac-bias-intrpt    = <0>;
-					dma-burst-sz      = <16>;
-					bpp               = <16>;
-					fdd               = <0x80>;
-					tft-alt-mode      = <0>;
-					stn-565-mode      = <0>;
-					mono-8bit-mode    = <0>;
-					sync-edge         = <0>;
-					sync-ctrl         = <1>;
-					raster-order      = <0>;
-					fifo-th           = <0>;
-				};
-				display-timings {
-					native-mode = <&timing1>;
-					timing0: 1024x768 {
-						hactive         = <1024>;
-						vactive         = <768>;
-						hback-porch     = <80>;
-						hfront-porch    = <48>;
-						vback-porch     = <15>;
-						vfront-porch    = <3>;
-						hsync-len       = <32>;
-						vsync-len       = <4>;
-						clock-frequency = <56000000>;
-						hsync-active	= <0>;
-						vsync-active	= <0>;
-					};
-					/* 1280 x 720 @ 60 Hz  Reduced blanking VESA CVT 0.92M9-R */
-					timing1: 1280x720-60 {
-						hactive         = <1280>;
-						hfront-porch    = <48>;
-						hsync-len       = <32>;
-						hback-porch     = <80>;
-						vactive         = <720>;
-						vfront-porch    = <3>;
-						vsync-len       = <5>;
-						vback-porch     = <13>;
-						clock-frequency = <64000000>;
-						hsync-active	= <0>;
-						vsync-active	= <0>;
-					};
-
-				};
+				i2c = <&i2c2>;
 			};
 
 			fb {
diff --git a/firmware/capes/cape-bone-dvi-00A2.dts b/firmware/capes/cape-bone-dvi-00A2.dts
index 1dbaef8..fb2dcf0 100644
--- a/firmware/capes/cape-bone-dvi-00A2.dts
+++ b/firmware/capes/cape-bone-dvi-00A2.dts
@@ -93,57 +93,13 @@
 				};
 			};
 
-			/* Settings for DVI 1024x768@60 */
+			/* use the DVI panel now */
 			panel {
-				compatible = "tilcdc,panel";
+				compatible = "tilcdc,tfp410";
 				pinctrl-names = "default";
 				pinctrl-0 = <&bone_dvi_cape_dvi_00A2_pins>;
 				ti,power-gpio = <&gpio1 31 0x0>;
-				panel-info {
-					ac-bias           = <255>;
-					ac-bias-intrpt    = <0>;
-					dma-burst-sz      = <16>;
-					bpp               = <16>;
-					fdd               = <0x80>;
-					tft-alt-mode      = <0>;
-					stn-565-mode      = <0>;
-					mono-8bit-mode    = <0>;
-					sync-edge         = <0>;
-					sync-ctrl         = <1>;
-					raster-order      = <0>;
-					fifo-th           = <0>;
-				};
-				display-timings {
-					native-mode = <&timing1>;
-					timing0: 1024x768 {
-						hactive         = <1024>;
-						vactive         = <768>;
-						hback-porch     = <80>;
-						hfront-porch    = <48>;
-						vback-porch     = <15>;
-						vfront-porch    = <3>;
-						hsync-len       = <32>;
-						vsync-len       = <4>;
-						clock-frequency = <56000000>;
-						hsync-active	= <0>;
-						vsync-active	= <0>;
-					};
-					/* 1280 x 720 @ 60 Hz  Reduced blanking VESA CVT 0.92M9-R */
-					timing1: 1280x720-60 {
-						hactive         = <1280>;
-						hfront-porch    = <48>;
-						hsync-len       = <32>;
-						hback-porch     = <80>;
-						vactive         = <720>;
-						vfront-porch    = <3>;
-						vsync-len       = <5>;
-						vback-porch     = <13>;
-						clock-frequency = <64000000>;
-						hsync-active	= <0>;
-						vsync-active	= <0>;
-					};
-
-				};
+				i2c = <&i2c2>;
 			};
 
 			fb {
-- 
1.7.10.4

